### PR TITLE
Fix/urdf to ignition

### PR DESCRIPTION
**Pull request by aldenpower**
### Description

This PR updates the description and control package to run the differential controller from ros2 control.

### Related Issue(s)

- N/A

### Checklist

- N/A

### CODE REVIEW

File: littlebot_control/launch/spawn_diffdriver_controller.launch.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import os
```
Comment (NestorDP) [inline littlebot_control/launch/spawn_diffdriver_controller.launch.py]: 'os' imported but unused
```suggestion

```

File: littlebot_control/launch/spawn_diffdriver_controller.launch.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+from launch_ros.actions import Node
+from launch import LaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
```
Comment (NestorDP) [inline littlebot_control/launch/spawn_diffdriver_controller.launch.py]: 'launch.launch_description_sources.PythonLaunchDescriptionSource' imported but unused
```suggestion

```

File: littlebot_control/launch/spawn_diffdriver_controller.launch.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+from launch.actions import RegisterEventHandler
+from launch.event_handlers import OnProcessExit
+
+
+def generate_launch_description():
+    joint_state_broadcaster_spawner = Node(
+        package="controller_manager",
+        executable="spawner",
+        arguments=["joint_state_broadcaster", "--controller-manager", "/controller_manager"],
```
Comment (NestorDP) [inline littlebot_control/launch/spawn_diffdriver_controller.launch.py]: line too long (93 > 79 characters)
```suggestion
        arguments=["joint_state_broadcaster",
                   "--controller-manager",
                   "/controller_manager"],
```

File: littlebot_control/launch/spawn_diffdriver_controller.launch.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    )
+
+    robot_controller_spawner = Node(
+        package="controller_manager",
+        executable="spawner",
+        arguments=["littlebot_velocity_controller", "--controller-manager", "/controller_manager"],
```
Comment (NestorDP) [inline littlebot_control/launch/spawn_diffdriver_controller.launch.py]: line too long (93 > 79 characters)
```suggestion
        arguments=["littlebot_velocity_controller",
                   "--controller-manager",
                   "/controller_manager"],
```

File: littlebot_control/launch/spawn_diffdriver_controller.launch.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        output="log",
+        # arguments=["-d", rviz_config_file],
+    )
+
+    delay_robot_controller_spawner_after_joint_state_broadcaster_spawner = RegisterEventHandler(
+        event_handler=OnProcessExit(
+            target_action=joint_state_broadcaster_spawner,
+            on_exit=[robot_controller_spawner],
+        )
+    )
```
Comment (NestorDP) [inline littlebot_control/launch/spawn_diffdriver_controller.launch.py]: line 30 too long (93 > 79 characters)

```suggestion
    delay_robot_controller_spawner_after_joint_state_broadcaster_spawner =\
        RegisterEventHandler(
            event_handler=OnProcessExit(
                target_action=joint_state_broadcaster_spawner,
                on_exit=[robot_controller_spawner],
            )
        )
```

File: littlebot_control/launch/spawn_diffdriver_controller.launch.py [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    delay_rviz_after_robot_controller = RegisterEventHandler(
+        event_handler=OnProcessExit(
+            target_action=robot_controller_spawner,
+            on_exit=[rviz_node],
+        )
+    )
+
+    return LaunchDescription([
+        # control_node,
```
Comment (NestorDP) [inline littlebot_control/launch/spawn_diffdriver_controller.launch.py]: Delete unecessary comment
```suggestion
```

**Review by NestorDP — COMMENTED** [review]
Please, run the Flake8 python lint before to open a new Pull Request

File: littlebot_control/launch/spawn_diffdriver_controller.launch.py (hunk 1 above)
Comment (aldenpower) [reply littlebot_control/launch/spawn_diffdriver_controller.launch.py]: fixed [here](https://github.com/NestorDP/littlebot/pull/10/commits/33db77a6b46130d550d8f299364a59b44734b9e4)

File: littlebot_control/launch/spawn_diffdriver_controller.launch.py (hunk 2 above)
Comment (aldenpower) [reply littlebot_control/launch/spawn_diffdriver_controller.launch.py]: fixed [here](https://github.com/NestorDP/littlebot/pull/10/commits/33db77a6b46130d550d8f299364a59b44734b9e4)

File: littlebot_control/launch/spawn_diffdriver_controller.launch.py (hunk 3 above)
Comment (aldenpower) [reply littlebot_control/launch/spawn_diffdriver_controller.launch.py]: fixed [here](https://github.com/NestorDP/littlebot/pull/10/commits/33db77a6b46130d550d8f299364a59b44734b9e4)

File: littlebot_control/launch/spawn_diffdriver_controller.launch.py (hunk 4 above)
Comment (aldenpower) [reply littlebot_control/launch/spawn_diffdriver_controller.launch.py]: fixed [here](https://github.com/NestorDP/littlebot/pull/10/commits/33db77a6b46130d550d8f299364a59b44734b9e4)

File: littlebot_control/launch/spawn_diffdriver_controller.launch.py (hunk 5 above)
Comment (aldenpower) [reply littlebot_control/launch/spawn_diffdriver_controller.launch.py]: fixed [here](https://github.com/NestorDP/littlebot/pull/10/commits/33db77a6b46130d550d8f299364a59b44734b9e4)

File: littlebot_control/launch/spawn_diffdriver_controller.launch.py (hunk 6 above)
Comment (aldenpower) [reply littlebot_control/launch/spawn_diffdriver_controller.launch.py]: fixed [here](https://github.com/NestorDP/littlebot/pull/10/commits/33db77a6b46130d550d8f299364a59b44734b9e4)

**Pull request merged**